book/website/collaboration/inclusive-events/inclusive-events-location.md

```diff
@@ -0,0 +1,249 @@
+and how far attendees will need to travel to get from the event location to their accommodation. 
+Many large venues have accommodation on-site and can be prioritized for attendees with lived experience such as those with disabilities or caring responsibilities.
+
+Check with a venue's child policy before booking and ensure that there are suitable arrangements made if a venue area does not allow children to be present. 
+For example, some museums and botanical garden greenhouses do not allow children to be present for private events. 
+This may mean that attendees with child-caring responsibilities are unable to attend and engage with your event. 
+
+It is also important to check that rooms, doors, restrooms, and all spaces in the venue has step-free access that is wide enough and are able to be accessed by attendees who may use mobility aids. 
+This can mean checking that ramps are no steeper than a 1:20 gradient, doors should have a minimum clear opening width of 840mm, and any space where attendees may need to change direction 
+has a 1.5m by 1.5m space which allows for most wheelchairs to turn in a complete circle *(cite: Portsmouth city council: https://www.portsmouth.gov.uk/wp-content/uploads/2020/05/development-and-planning-design-guide-for-wheelchair-accessible-housing.pdf)*. 
+Many motorized wheelchairs may be larger and require additional space to travel and turn.
+
+Note the location of access ramps and assess if you will need to provide additional signage to guide attendees who use the ramps to access the event location. 
+Signs should be viewable from a distance and clearly legible.
+
+Ensure that the venue has seating that is suitable for attendees who may use mobility aids and wheelchairs, as well as seating for low-vision and low-hearing attendees. 
+Notice if this seating is significantly segregated and how you will make attendees feel welcome and included at the event. 
+
+Depending on the size of your venue, you may consider if there is space available to use as a multi-faith prayer room or a quiet room that reserves a space away from the event to relax, 
+take medicine privately, and take a sensory break for those who need it. These spaces can help make an event more inclusive and enjoyable for your attendees. 
+
+When reviewing potential venues, consider the ventilation of airflow, lighting and acoustics, and ensure that they are appropriate for the number of attendees at your event. 
+A large hall with high ceilings may echo and not be suitable for discussion-based sessions, and a small room with limited ventilation may not be suited for long sessions with a large number of attendees. 
+
+COVID-19 highlighted the importance of ventilation for the safety and health of event attendees, try to ensure that there is adequate airflow at your venue 
+and discuss how you will ensure the safety of attendees who may be immunocompromised. 
```

Comment:
I might leave it separate to try make it easier to read (instead of a large block) and so that comments about covid aren't lost in the middle of a paragraph. But I do see your point. I think for the sake of reading and emphasis I will leave it as it.